src/utils/chain.ts

```diff
@@ -0,0 +1,10 @@
+export const EIPNumberPrefix = 'eip155:'
+
+/**
+ * Returns the chain string combined with EIP number with the given chainId
+ * @param chainId - chain number
+ * @returns string
+ */
+export const getChainString = (chainId?: number) => {
+  return EIPNumberPrefix.concat(chainId?.toString() || '')
+}
```

Comment:
We already have `getChain` function in `utils/address`

Reply:
Seems like `getChain` and `getChainString` does different things. But just to disallow confliction, updated the method name as `getEIPChainString`, wdyt?